content/guides/understanding-inbox-messaging.md

```diff
@@ -0,0 +1,77 @@
+how to implement this integration.
+
+
+## Purpose
+
+The purpose of this feature is to provide a seamless integration between internal messaging systems in certain Channels and the BookingSync platform, in order for adminstrators to be able to handle incoming inquiries in a centralized location.
+
+
+## Entities
+
+- **Inbox Conversation**: Represents a thread/conversation and is scoped under a Channel, an Account, and a Booking.
+<br>
+- **Inbox Participant**: Represents a participant in the thread and is scoped under an Inbox Conversation. There are two types of participants
+  - Client: Represents a Traveller, managed by the Channel entirely.
+  - Host: Represents a User in the Inbox Conversation's Account, managed by the Channel and the Account. All Users should be able to view the Inbox Conversations of their corresponding Account. However, Only Host Inbox Participants could send messages and be notified about new messages in the Inbox Conversation.
+<br>
+- **Inbox Message**: Represents a message in the thread/conversation and is scoped and an Inbox Participant.
+
+
+## Flow
+
+When a Booking is created on the following channels: (Booking.com, Airbnb), a corresponding Inbox Conversation is created on our system, which is synchronised with the Channel internal messaging systems.
+
+When an Inbox Conversation is created via a Channel, at least one Client Inbox Participant will be added to the conversation automatically. A Host Inbox Participant may or may no be created via the Channel automatically.
+
+
+## Fetching an Inbox Conversation
+
+There are multiple ways to fetch an existing Inbox Conversation.
+
+1. Using the [Inbox Conversations endpoints](/reference/endpoints/inbox_conversations), you could get all the Inbox Conversations related to your account.
+2. Using the [Bookings endpoints](/reference/endpoints/application) `conversations` field, which you could then query the [Inbox Conversations endpoints](/reference/endpoints/inbox_conversations) with the provided ids to get Inbox Conversations related to a certain Booking.
+
+In order to discern which Channel this Inbox Conversation is associated with, you can check `created_by_id` and `created_by_type` attributes of the Inbox Conversation. `created_by_type` should be `Doorkeeper::Application` and `created_by_id` should refer to the Application id. Applications can be identified in [Applications endpoint](/reference/endpoints/applications).
+
+It is important to take into account that Inbox Conversations may not be created right away with the Booking, as there could be a delay. It can be beneficial to subscribe to `inbox_conversation_created` webhooks (Refer to [Webhook subscriptions guide](/guides/webhook-subscriptions)) and/or have periodic polling on the Booking to be notified when the associated Inbox Conversation is created.
+
+---
+
+***Note:*** Please refrain from creating a new Inbox Conversation as it will not be synchronised to the Channel's messaging system.
```

Comment:
```suggestion
***Note:*** Please refrain from creating a new Inbox Conversation as it will not be synchronized to the Channel's messaging system.
```

Reply:
🇺🇸 / 🇬🇧 not sure if we have a strict standard on that